src/util/serverUtils.js

```diff
@@ -0,0 +1,72 @@
+import https from 'https';
```

Comment:
moved these functions from `src/server.js` to this utils module to make it easier to mock them when testing the `server.js` code